src/WebSdk/Publish/Targets/Microsoft.NET.Sdk.Publish.targets

```diff
@@ -146,6 +147,14 @@ Copyright (C) Microsoft Corporation. All rights reserved.
 
   <Import Project="$(_DotNetCLIToolTargetsDir)Microsoft.NET.Sdk.DotNetCLITool.targets" Condition="Exists('$(_DotNetCLIToolTargetsDir)Microsoft.NET.Sdk.DotNetCLITool.targets')"/>
 
+<!--
+  ***********************************************************************************************
+  Import the Containers target
+  ***********************************************************************************************
+ -->
+
+  <Import Project="$(_ContainersTargetsDir)Microsoft.NET.Build.Containers.targets" Condition="Exists('$(_ContainersTargetsDir)Microsoft.NET.Build.Containers.targets') AND '$(EnableSdkContainerSupport)' == 'true'" />
```

Comment:
super-nit: Same as in `.props`, tiny bit better to try to short-circuit on the property first.